_publications/jors/paper.tex

```diff
@@ -0,0 +1,362 @@
+    CoffeeScript     & 1.12.7  \\
+    CSS              & 4       \\
+    \hline
+  \end{tabular}
+  \label{tab:versions}
+\end{table}
+
+
+\section*{Additional system requirements}
+
+There are no additional system requirements.
+
+\section*{Dependencies}
+
+The entire environment can be built using the Nix Package Manager so
+the only required dependency is a functional Nix installation. The
+PFHub framework has over 2000 separate package dependencies using
+data from the Nix package manager. The full dependency graph for PFHub
+can be seen online~\cite{dependencies}.
+
+\section*{List of contributors}
+
+1. Wheeler, Daniel; A, \githublink{wd15} \\
+2. Keller, Trevor; A, \githublink{tkphd} \\
+3. DeWitt, Stephen J.; E, \githublink{stvdwtt} \\
+4. Jokisaari, Andrea M.; D, \githublink{amjokisaari} \\
+5. Schwen, Daniel; D, \githublink{dschwen} \\
+6. Guyer, Jonathan E.; A, \githublink{guyer} \\
+
+See the contributors list on GitHub~\cite{contributors}.
+
+\section*{Software location:}
+
+{\bf Archive}
+
+\begin{description}[noitemsep,topsep=0pt]
+	\item[Name:] Zenodo
+	\item[Persistent identifier:]
+          \href{https://dx.doi.org/10.5281/zenodo.2592705}{10.5281/zenodo.2592705}
+	\item[Licence:] NIST Software License~\cite{nistlicense}
+	\item[Publisher:]  Daniel Wheeler
+	\item[Version published:] v0.1
+	\item[Date published:] 13/03/19
+\end{description}
+
+{\bf Code repository}
+
+\begin{description}[noitemsep,topsep=0pt]
+	\item[Name:] GitHub
+	\item[Persistent identifier:] \url{https://github.com/usnistgov/pfhub/tree/v0.1}
+	\item[Licence:] NIST Software License~\cite{nistlicense}
+	\item[Date published:] 13/03/19
+\end{description}
+
+\section*{Languages}
+
+English
+
+\section*{(3) Reuse potential}
+
+The website infrastructure can be cloned as a Git repository or
+downloaded as a ZIP archive and deployed with minimum effort. The
+mechanism for uploading data using Staticman can be easily configured
+for a new repository location. However, customizing the content of the
+website for a particular scientific community would require
+considerable effort. The following steps are the more challenging
+aspects of deploying the framework for a new community.
+
+\begin{itemize}
+  \item Upload new benchmark definitions in a format that Jekyll can
+    parse, \emph{e.g.}, Jupyter Notebook, Markdown or HTML.
+  \item Edit the \texttt{benchmarks.yaml} file to reflect the new
+    upload requirements and describe the figures that need to be
+    generated on the upload comparison pages.
+  \item Edit the \texttt{\_config.yml} file to update links and text
+    related to the configuration for all aspects of the website.
+  \item Update Markdown files to reflect the new content and mission
+    of the scientific community.
+  \item Remove data and files that are not required by the new
+    community.
+\end{itemize}
+
+Currently, a deployment for a new community has not been attempted
+and, thus, the above steps need to be refined and documented.
+
```

Comment:
Perhaps it's not a good idea, but I'm wondering if we should add some kind of section that discusses the success of the website.  Usage statistics, uploads, screenshots of the website...  Some kind of walk-through example so that the software framework described actually has a concrete example to the reader.

Reply:
I've now mentioned that we have 108 separate benchmark uploads in the Implementation and Architecture section. Regarding the statistics, GitHub records a lot of that about the code, downloads and forks etc. I think it's clear to most readers that they can go to GitHub for that sort of meta-data.

I'm not sure what to do for the walk-through example. We already have the workflow outline in the second paragraph of the Implementation section. Also the paper is getting quite long for its purpose. Reviewers / readers can always go to the website and submit model data. I'll point that out in the cover letter with the paper. I'll await your thoughts on this before proceeding.

Reply:
I understand from your comments more about the publishing requirements for the intended journal.  As such, I think your changes are good and suggestions for the cover letter.  I wouldn't add anything more to the paper.